static/app/views/nav/primary/help.tsx

```diff
@@ -55,10 +56,12 @@ export function PrimaryNavigationHelp() {
   const contactSupportItem = getContactSupportItem({organization});
   const openForm = useFeedbackForm();
   const {startTour} = useStackedNavigationTour();
+  const chonkPrompt = useChonkPrompt();
 
   return (
     <StackedNavigationTourReminder>
       <SidebarMenu
+        onClick={chonkPrompt.dismissDotIndicatorPrompt}
```

Comment:
I think we should call this `onItemClick?` or something, as it gets triggered when an item is clicked, not when the `SideBarMenu` is clicked

Reply:
Is it `onAction` that you are referring to here? The different namings of these callbacks are such PITA, because they differ between each component (I could have sworn I looked over the type to see if there is anything)

Reply:
Agh, actually onAction is just for individual actions. I don't think this component supports anything like what you are referring to actually :/